tests/solvers/restart_01.cc

```diff
@@ -164,7 +164,7 @@ test()
   NSparam.restart_parameters.frequency  = 1;
   NSparam.non_linear_solver.verbosity   = Parameters::Verbosity::quiet;
   NSparam.linear_solver.verbosity       = Parameters::Verbosity::quiet;
-  NSparam.boundary_conditions.createDefaultNoSlip();
+  NSparam.boundary_conditions.createNoSlip();
```

Comment:
Is NoSlip the boundary condition created by default or is it a none now? If it is still no Slip, then that's ok. Just to verify.

Reply:
It is `none`! In fact this `createDefaultNoSlip` is only used in this test, still I had to keep the method so that the test is not broken. I renamed it `createNoSlip` to clarify that this is not the case by default anymore :)

Reply:
Ohh it's just in a test. I see.
Ready fo rmergsaurus rex then.